apps/galleries_institutions/components/filter_facet/locationSynonyms.coffee

```diff
@@ -0,0 +1,4 @@
+module.exports = [
+  ['st', 'saint']
```

Comment:
Maybe changing the implementation to use an object/dictionary (constant lookup time) may speed up the matching? If we have unique keys, we can have something like:
```javascript
{
  saint: 'st',
  harbour: 'harbor'
}
```